src/main/java/com/easypost/service/EasyPostClient.java

```diff
@@ -6,11 +6,11 @@
 public class EasyPostClient {
     private final int connectTimeoutMilliseconds;
     private final int readTimeoutMilliseconds;
-    private final String apiKey;
+    private final String inputApiKey; // API key for all EasyPost API requests
```

Comment:
User will never see the `inputApiKey` because it's private and for internal use only. Why do we want to add another breaking change because of the namespace that is being used internally only? We could rename the `inputApiKey` to a better name.

Reply:
OH! I misunderstood how this worked. I thought THIS was the name of the service and I didn't want users to need to do `client.inputApiKey.all()` heh.

Yes, this is fine, though let's call it `userApiKey` or `clientApiKey` to be explicit 🙂